tests/lib/json/src/main.c

```diff
@@ -39,6 +39,12 @@ struct obj_array {
 	size_t num_elements;
 };
 
+struct test_int_limits {
+	int int_max;
+	int int_cero;
```

Comment:
Spanish? I don't really mind as long as it's not a public API ;)

Reply:
Thanks so much for the quick review! Nice catch, the c and z will be the end of me 😄. Just in the internal testing, I'll fix it on my next PR :)